interpreter/cling/include/cling/Interpreter/Transaction.h

```diff
@@ -316,6 +322,14 @@ namespace cling {
       m_Opts = CO;
     }
 
+    clang::NamespaceDecl* getDefinitionNS() const { return m_DefinitionNS; }
```

Comment:
Can you add an empty line after the function?
```suggestion
    clang::NamespaceDecl* getDefinitionNS() const { return m_DefinitionNS; }

```